### ISSUE TITLE
does the world need blueprints when it has Hygen?

### ISSUE THREAD

**Issue by anithri** [SpencerCDixon/redux-cli#136]
I encountered a project called [Hygen](http://www.hygen.io/), which is very similar to blueprint/redux-cli.  Not in a stole the code or anything.  But the way the templates and cli work.  But with all of the changes I was hoping to get into this project.
1. Generators to create generators
2. Template files use FrontMatter to define template data.  the `to`: field is the destination.
3. Inject data into files (routes, dependencies)
4. cli args to pass data to templates
5. Multiple locations for templates
6. build prompts to ask user for data
7. Customization data sources possible

It does everything I need a generator tool to do, and it would take me very little time to port my existing blueprints to templates and generators.

So to be blunt, does the world need blueprints when it has hygen?  Is there value to continue efforts here?

@SpencerCDixon @jamesr73 @walreyes

**Comment by anithri** [comment]
I wanted to improve a tool I use a lot, and now Hygen fits the bill. I can't really see a reason to continue working on this. I'd rather spend my meager open source time scratching other itches. 

I will absolutely help get this into shape for someone else to take over, or for the 2.0 work we started to continue.  I'd hate to think I left you in the lurch.

**Comment by SpencerCDixon** [comment]
Alllll good man don't worry about it.  Thanks for the time and effort you've put into it thus far!  Cheers 🍻

**Comment by walreyes** [comment]
It's all good! I didn't put that much effort but I learned a lot a long the way!

I actually posted this [blog post](https://medium.com/sudo-by-icalia-labs/libraries-to-build-a-strong-and-beautiful-cli-tool-with-javascript-b753ff0b0556) from what a I learned digging into this repo:

**Comment by jamesr73** [comment]
Be free. Be happy. I had fun helping out but then I got a new job that meant this wasn’t too important for me anymore. I’ll take a look at Hygen though! :-)

**Comment by walreyes** [comment]
Good vibes 👍

**Comment by tomlagier** [comment]
Much respect for noting an overlap of effort and promoting the project that you think has everything you want in it. I really appreciate that over just silently abandoning a fairly well-used repo.

Is there any way we could maybe get that codified in the readme? This repo has a lot of SEO visibility, would be great to link Hygen if that's the recommended tool here.

**Comment by JawsomeJason** [comment]
+1 on updating the README at the beginning of the file.  I spent 20 minutes getting this setup only to switch to Hygen.

**Comment by nsatija** [comment]
I understand that there is Hygen but this works great too at least for me and discouraging its development isn't benefitting anyone. Good work @SpencerCDixon and Thank You

**Comment by JawsomeJason** [comment]
@nsatija No one is discouraging development.  There has just been no recent work done on it, and there are other tools that are actively developing. It benefits no one to not include other similar tools.

**Comment by nsatija** [comment]
@thejase I agree with you. I would love it if @SpencerCDixon found the time and motivation to develop it further. In the mean time i did try hygen with npm global install not brew install and it errored for me :(
So I am using https://www.npmjs.com/package/react-gc now but what I like about redux-cli was generating  "ducks". I like generating my ducks in a row :)

**Comment by walreyes** [comment]
This is a great community! Taking the time to discuss this kind of things is awesome 👌 

If you get in a new project or you guys decide to keep developing `redux-cli` I'd love to keep collaborating.

**Comment by easingthemes** [comment]
I don't see any reasons to keep this still open. 
Also there was no reason to open it at the first place.
There are millions of similar tools, like everything else in FE world.

I was using this tool for years, always worked great, and still works great.

```
does the world need Grunt when it has Gulp?
does the world need Gulp when it has npm?
does the world need npm when it has yarn?
does the world need Sass when it has Less?
does the world need Angular when it has React?
does the world need React when it has Vue?
...
```
